src/components/App/ModalApp.tsx

```diff
@@ -214,21 +255,4 @@ class ModalApp extends Component {
   }
 }
 
-const mapStateToProps = (state) => ({
-  ...state.globals,
-  captures: state.captures,
-})
-
-const mapDispatchToProps = (dispatch) => ({
-  actions: bindActionCreators(actions, dispatch),
-})
-
-const ConnectedModalApp = connect(mapStateToProps, mapDispatchToProps)(ModalApp)
-
-const App = ({ options }) => (
-  <ReduxAppWrapper options={options}>
-    <ConnectedModalApp options={options} />
```

Comment:
`<ModalApp />` declares 2 props: `options: SdkOptions` and `socket?: SocketIOClient.Socket` but I can't find anywhere to use the second one. Was it removed already?